src/elements/content-sidebar/__tests__/__snapshots__/SidebarNav-test.js.snap

```diff
@@ -1,122 +1,265 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`elements/content-sidebar/SidebarNav should render the additional tabs loading state 1`] = `
-<SidebarNav
-  additionalTabs={Array []}
-  hasAdditionalTabs={true}
->
-  <nav>
-    <AdditionalTabs
-      tabs={Array []}
+Array [
```

Comment:
Is this array coming from the `injectIntl` mock in `react-intl-mocks.js`? It seems like it should just be a decorator/passthrough.

Reply:
Yeah, this was a mistake, due to the new intl mock it's adding a component around SidebarNav with the same name, so enzyme is finding two `SidebarNav` components. I'll update the test to grab the real one